### PR TITLE
Slb/ad/cadenza 38410

### PR DESCRIPTION
**Pull request by qulol**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
+### Fixed
+- Removed duplicate geometry section in `sandbox.html#editGeometry`
+- Horizontal scrollbar in `sandbox.html` left sidebar now appears sticky.
```
Comment (jkissel) [inline CHANGELOG.md:10]: A Horizontal scrollbar is rather painful to use. Why do we want it here?

**Review by jkissel — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (qulol) [reply CHANGELOG.md:10]: We currently have it due to long explaination texts, and this is a improvement to the current state imo.

Comment (jkissel) [reply CHANGELOG.md:10]: But the sidebar is fixed width. We should be able to craft the contents in a way that does not require a horizontal scrollbar. For example texts should wrap lines. We can have a look if necessary.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 This project uses a version scheme based on the Cadenza main version in the format x.x.y, where x.x is the Cadenza main version and y a functional change or bugfix.
 
 ## Unreleased
+### Fixed
+- Removed duplicate geometry section in `sandbox.html#editGeometry`
+
+### Changed
+- Examples are now displayed more compact
```
Comment (klschm) [inline CHANGELOG.md:12]: - Examples in the sandbox now are displayed more compact

**Review by klschm — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (jkissel) [reply CHANGELOG.md:12]: Or:
> Improved the examples in the sandbox to avoid a horizontal scrollbar.

**Review by jkissel — APPROVED** [review]

**Pull request merged**